docs/requirement.txt

```diff
@@ -0,0 +1,3 @@
+sphinx
+sphinx-rtd-theme
+sphinxcontrib-napoleon
```

Comment:
Is it used by sphinx or meant to be information for user? If the later, this is not consistent with "pip install sphinx sphinx-autoapi" in the workflow